webapp/graphite/remote_storage.py

```diff
@@ -78,7 +78,7 @@ def get_results(self):
       self.send()
 
     try:
-      response = self.connection.getresponse()
+      response = self.connection.getresponse(buffering=True)
```

Comment:
For supporting python 2.6 it can be rewritten like
```
    try:
      response = self.connection.getresponse(buffering=True)
    except TypeError:
      response = self.connection.getresponse()

    try:
      assert response.status == 200, "received error response %s - %s" % (response.status, response.reason)
...
```